### ISSUE TITLE
Letsencrypt does not provide correct cert path to Installer plugins

### ISSUE THREAD

**Issue by e00E** [letsencrypt/letsencrypt#472]
In deploy_cert(domain, cert_path, key_path, chain_path=None) the cert_path does not lead to a file.
I am on debian unstable and the passed cert_path is:
/etc/letsencrypt/certs/cert-letsencrypt.pem
However that file does not exist. 
```
ls /etc/letsencrypt/certs/:
0000_cert-letsencrypt.pem  0000_csr-letsencrypt.pem  0001_cert-letsencrypt.pem  0001_csr-letsencrypt.pem  0002_csr-letsencrypt.pem
```
The path to the private key seems correct.

**Comment by kuba** [comment]
This is all scraped in #440, where `--cert-path` and `--chain-path` are flags for `install` and `revoke` sub commands only. See https://github.com/letsencrypt/lets-encrypt-preview/pull/440/files#diff-54b2f468d57ef7436008bd031bc89711R315. `required=True` makes sure that user supplies his own value, which is expected to be a path to a existing file with valid certificate. `/etc/letsencrypt/certs/cert-letsencrypt.pem` is not used anymore.

**Comment by kuba** [comment]
This should be fixed as of #558.
